requirements.txt

```diff
@@ -1,18 +1,17 @@
-appdirs==1.4.4
-networkx==2.6.3
-numba==0.53.1
-numpy==1.19.2
-plotly==4.4.1
-python-dateutil==2.8.0
+networkx==2.7.1
+numba==0.55.1
+numpy==1.21.5
+plotly==5.6.0
+python-dateutil==2.8.2
 quantum-blackbird==0.4.0
-requests==2.22.0
-scipy==1.7.1
-sympy==1.8
-tensorflow==2.6.3
-tensorboard==2.6.0
-thewalrus==0.18.0
+requests==2.27.1
+scipy==1.8.0
+sympy==1.10
+tensorflow==2.8.0
+tensorboard==2.8.0
+git+https://github.com/XanaduAI/thewalrus.git@master#thewalrus
```

Comment:
Installs The Walrus from master. Will update before release.

Reply:
Tests will fail until [#337](https://github.com/XanaduAI/thewalrus/pull/337) is merged in The Walrus. They've been tested using the PR branch before and pass.